.eleventy.js

```diff
@@ -279,6 +280,45 @@ module.exports = function (config) {
     }
   }
 
+  if (process.env.ALT_BUILD) {
+    config.addCollection('courses', (collections) => {
+      const coursesGlob = '**/en/learn/*/*.md';
+      collections.getFilteredByGlob(coursesGlob).forEach(async (course) => {
```

Comment:
This could lead to unfulfiled promises, respectively unwritten files as it's not awaited higher up I think.

Reply:
I added a warning comment